src/components/icons/IAIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import { SVGProps } from "react";
+
+export const IAIcon = (props: SVGProps<SVGSVGElement>) => (
```

Comment:
Imo this is just way easier to keep track of

```suggestion
export const IAIcon = (props: ComponentProps<"svg">) => (
```